docs/sphinx/developers/cpp/tutorial.txt

```diff
@@ -493,3 +525,59 @@ Full example source: :download:`metadata-formatreader.cpp <examples/metadata-for
   - :doxygen:`FormatReader <classome_1_1bioformats_1_1FormatReader.html>`
   - :doxygen:`TIFFReader <classome_1_1bioformats_1_1in_1_1TIFFReader.html>`
   - :doxygen:`OMETIFFReader <classome_1_1bioformats_1_1in_1_1OMETIFFReader.html>`
+
+Writing images
+--------------
+
+Image writing is performed using the :cpp:class:`FormatWriter`
+interface.  This is an abstract writer interface implemented by
+file-format-specific writer classes.  Examples of writers include
+:cpp:class:`MinimalTIFFWriter`, which implements writing of Baseline
+TIFF and :cpp:class:`OMETIFFWriter` which implements writing of
+OME-TIFF (TIFF with OME-XML metadata).
+
+Using a writer involves these steps:
+
+#. Create a writer instance
+#. Set metadata store to use
+#. Set options to control writer behavior
+#. Call :cpp:func:`setId` to specify the image file to write
+#. Store pixel data for each plane of each image in the specified
+   dimension order
```

Comment:
I don't know; I thought we omitted them for bulleted and numbered lists.  @hflynn what's the policy here?
This is matching the list in the reader section, above, but I can change both if it's wrong.